CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ All notable changes to this project will be documented in this file.
 -   \[DEVELOPER\] Added an improved traceback to the developer-mode errors for errors in function calls.
 
 ### Fixed
-
+-   #1820 : Deprecated unused decorator `@lambdify`
```

Comment:
I think this should go in the "Deprecated" section below. The space between the subtitle and the list should also be kept. I'm not sure when it was accidentally dropped in the "Added" and "Changed" sections for this release. I will put it back in another PR.